components/Dashboard/MyFeed/MyFeed.tsx

```diff
@@ -91,7 +110,9 @@ const MyFeed: React.FC<Props> = ({ currentUser }) => {
       <Head>
         <title>{pageTitle}</title>
       </Head>
-      <h1>My Feed</h1>
+      <h1>
+        {greetings[greetingLanguage]} {currentUser.name || currentUser.handle}!
```

Comment:
@Lanny hmm, TS shouts at me here because of an implicit `any` type, even though it seems to clearly know the type... 🤔 Tried a few annotations but I seem to be doing something wrong

![image](https://user-images.githubusercontent.com/34203886/93374359-73d6b100-f80b-11ea-8952-6f18cd218bf6.png)

Reply:
Fixed this now 🙂 ✅